strr-api/src/strr_api/services/approval_service.py

```diff
@@ -214,8 +213,7 @@ def process_auto_approval(cls, token, application: Application) -> Tuple[Any, An
                             registration = RegistrationService.create_registration(
                                 application.submitter_id, application.payment_account, registration_request.registration
                             )
-                            registration_status = RegistrationStatus.PROVISIONAL
-                            registration.status = registration_status
+                            registration.status = RegistrationStatus.ACTIVE
```

Comment:
I don't setting the status is needed now as it will be set to active when registration is created.